aspnetcore/mvc/views/view-components.md

```diff
@@ -82,6 +82,14 @@ The default view name for a view component is *Default*, which means your view f
 
 We recommend you name the view file *Default.cshtml* and use the *Views/Shared/Components/{View Component Name}/{View Name}* path. The `PriorityList` view component used in this sample uses *Views/Shared/Components/PriorityList/Default.cshtml* for the view component view.
 
+### Customizing the view search path
```

Comment:
```suggestion
### Customize the view search path
```

Reply:
Thanks @scottaddie. I started with this and changed my mind. I felt like that made it sound more like an instruction rather than a "this is how you do it". The next section uses "Invoking" over "Invoke" too.

I'll go with your suggestion though. Just explaining my thought process.

Reply:
Not your fault. The heading below violates the style guide too. The rule of thumb is to avoid using gerunds in headings. We can fix that up the next time this doc is updated.